adagucserverEC/CServerParams.cpp

```diff
@@ -543,10 +543,16 @@ bool CServerParams::checkTimeFormat(CT::string& timeToCheck){
 
 int CServerParams::parseConfigFile(CT::string &pszConfigFile){
   CT::string configFileData;
-  
+  CT::string subStringOfConfigString = pszConfigFile;
```

Comment:
Hier zou van mij wel wat commentaar bij mogen.. 
Het lijkt mij dat dit ook een wijziging in het kader van het ophalen van de config file uit de database is.